cluster/Vagrantfile

```diff
@@ -96,7 +96,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           c.vm.provision :shell, path: 'bootstrap_centos.sh'
           if role == 'master'
             # Install kubernetes on master
-            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.4.7'
+            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.5.2'
```

Comment:
Why is this needed? Should we not stick to same version as supported matrix for now?